spot_driver/spot_driver/spot_ros2.py

```diff
@@ -2919,6 +2925,35 @@ def handle_set_gripper_camera_parameters(
 
         return response
 
+    def set_arm_velocity_callback(self, data: ArmVelocityCommandRequest) -> None:
```

Comment:
```suggestion
    def handle_arm_velocity_command(self, arm_velocity_command: ArmVelocityCommandRequest) :
```

Reply:
Yeah sure, why not, however the creation of the subscription needs also to be changed. will fix it